tests/sema/Makefile

```diff
@@ -3,5 +3,7 @@ include ../Makefile.tests_common
 USEMODULE += sema
 USEMODULE += xtimer
 USEMODULE += ztimer_usec
+USEMODULE += ztimer_xtimer_compat
+USEMODULE += ztimer64_xtimer_compat
```

Comment:
Why do we need to add ztimer_xtimer_compat and ztimer64_xtimer_compat (shouldn't ztimer64_xtimer_compat depend on ztimer_xtimer_compat?)?